apps/external-contracts-tests/test-suites/national-registry.spec.ts

```diff
@@ -0,0 +1,25 @@
+// NOTE: To run this locally, you'll need to portforward soffia and set
+// the environment variable "SOFFIA_SOAP_URL" to https://localhost:8443
+// kubectl port-forward svc/socat-soffia 8443:443 -n socat
+import { NationalRegistryApi } from '@island.is/clients/national-registry-v1'
+
+describe('is.island.external.national', () => {
+  let client: NationalRegistryApi
+  beforeAll(async () => {
+    client = await NationalRegistryApi.instantiateClass({
+      baseSoapUrl: process.env.SOFFIA_SOAP_URL!,
+      user: process.env.SOFFIA_USER!,
+      password: process.env.SOFFIA_PASS!,
+      host: process.env.SOFFIA_HOST_URL!,
+    })
```

Comment:
Would be great to update this with the new NationalRegistry client and use the NestJS DI to construct and get the service instance. This way it is configured exactly the same way as in other apps.

We can either use the same `Test.createTestingModule()` setup as we use in other jest tests, or have a shared AppModule that contains everything which all the tests need.

Reply:
Done. Although I am getting 401 on both Staging and Dev.